test/integration/smoke/test_storage_policy.py

```diff
@@ -0,0 +1,263 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+""" Test for VMWare storage policies
+"""
+# Import Local Modules
+from nose.plugins.attrib import attr
+from marvin.cloudstackTestCase import cloudstackTestCase
+from marvin.lib.utils import cleanup_resources
+from marvin.lib.base import (Account,
+                             VirtualMachine,
+                             ServiceOffering,
+                             NetworkOffering,
+                             Network,
+                             Volume,
+                             DiskOffering)
+from marvin.lib.common import (get_zone,
+                               get_domain,
+                               get_test_template)
+from marvin.codes import PASS
+from marvin.cloudstackAPI import (importVsphereStoragePolicies)
+from marvin.cloudstackAPI import (listVsphereStoragePolicies)
+
+
+class TestVMWareStoragePolicies(cloudstackTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+
+        cls.testClient = super(TestVMWareStoragePolicies, cls).getClsTestClient()
+        cls.apiclient = cls.testClient.getApiClient()
+
+        cls.testdata = cls.testClient.getParsedTestDataConfig()
+        # Get Zone, Domain and templates
+        cls.domain = get_domain(cls.apiclient)
+        cls.zone = get_zone(cls.apiclient, cls.testClient.getZoneForTests())
+        cls._cleanup = []
+        cls.hypervisor = cls.testClient.getHypervisorInfo()
+        cls.network_offering = NetworkOffering.create(
+            cls.apiclient,
+            cls.testdata["l2-network_offering"],
+        )
+        cls.network_offering.update(cls.apiclient, state='Enabled')
+        cls.template = get_test_template(
+            cls.apiclient,
+            cls.zone.id,
+            cls.hypervisor,
+        )
+        cls.cleanup = [cls.network_offering]
```

Comment:
this will not be cleaned, needs to be something like below
```suggestion
        cls._cleanup.append(cls.network_offering)
```